### PR TITLE
UX: add crown

### PR DESCRIPTION
**Pull request by chapoi**
Simply added a crown to the #1 go-getter

image

### CODE REVIEW

File: assets/javascripts/discourse/components/minimal-gamification-leaderboard.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ export default class extends Component {
       if (user.id === this.model.personal?.user?.id) {
         user.isCurrentUser = "true";
       }
+      if (user.id === 1) {
```
Comment (CvX) [inline assets/javascripts/discourse/components/minimal-gamification-leaderboard.js:33]: id? 😃

Comment (chapoi) [reply assets/javascripts/discourse/components/minimal-gamification-leaderboard.js:33]: it looked weird to me too, prolly didnt understand it fully… what should I use then?

Comment (chapoi) [reply assets/javascripts/discourse/components/minimal-gamification-leaderboard.js:33]: ok wait I know I think

Comment (chapoi) [reply assets/javascripts/discourse/components/minimal-gamification-leaderboard.js:33]: K, nvm I dont

**Review by CvX — APPROVED** [review]

**Pull request merged**